config/config.env

```diff
@@ -119,6 +119,12 @@ S3_GW_CORS_DEFAULT_MAX_AGE=600
 # If not set, default value 0 will be used -- it means that object will be processed according to the container's placement policy
 S3_GW_NEOFS_SET_COPIES_NUMBER=0
 
+# Timeout between retrieving actual epoch from NeoFS. Actual only if slicer.enabled = true.
+S3_GW_NEOFS_EPOCH_UPDATE_INTERVAL=30s
```

Comment:
Seems to be a bit aggressive.

Reply:
Replaced with 2m. The value I got after epoch_duration and ms_per_block math.